add shadow wrangler event role (#1667)
writethedocs/www: docs/organizer-guide/confs/event-roles.rst

    and the process was streamlined. It's still time-consuming and intense, 
    because we try to decide pretty quickly after the CFP closes.

> * **Shadow wrangler**

Reviewer: ```suggestion
* **Shadowing coordinator**
```

Reviewer: We try to avoid using "wrangler" in public-facing content :)

Author: ah, OK! I was indeed unsure about referring to a "wrangler" here. We make an exception for "Speaker Wrangler", though?

  * This role is similar to the volunteer coordinator role.

>   * It is somewhat specific to the context of our remote conferences, which have made it challenging to recruit,

Reviewer: ```suggestion
  * It is more relevant in the context of our remote conferences, which have made it challenging to recruit,
```

    onboard, and retain new volunteers.

>   * This person recruits volunteers who are interested and committed to supporting the organizing team.

Reviewer: ```suggestion
  * This person contacts volunteers who are interested and committed to supporting the organizing team, based on recommendations from other organizers or out of the pool of previous volunteers and speakers.
```

>   * They ask volunteers to "shadow" organizers in order to learn how to perform their roles and step-in as needed.

Reviewer: ```suggestion
  * The coordinator then matches volunteers with organizers to "shadow" in order to learn how to perform their roles and step-in as needed. 
  * The organizers are responsible for training the shadowing volunteer on the specific responsibilities, but they can request help with logistics and administration from the coordinator for things like document permissions and registrations.
```
